docs/loading/Spark-connector-starrocks.md

```diff
@@ -163,15 +165,42 @@ Construct data in memory and load data into the StarRocks table.
     // 2. Write to starrocks with the format "starrocks",
     // and replace the options with your own.
     df.write.format("starrocks")
-        .option("starrocks.fe.http.url", "127.0.0.1:8038")
-        .option("starrocks.fe.jdbc.url", "jdbc:mysql://127.0.0.1:9038")
+        .option("starrocks.fe.http.url", "127.0.0.1:8030")
+        .option("starrocks.fe.jdbc.url", "jdbc:mysql://127.0.0.1:9030")
         .option("starrocks.table.identifier", "test.score_board")
         .option("starrocks.user", "root")
         .option("starrocks.password", "")
         .mode("append")
         .save()
     ```
 
+For python, run the following codes in `pyspark`:
+
+   ```python
+   from pyspark.sql import SparkSession
+
+   spark = SparkSession \
+        .builder \
+        .appName("StarRocks Example") \
+        .getOrCreate()
+
+    # 1. Create a DataFrame from a sequence.
+    data = [(1, "starrocks", 100), (2, "spark", 100)]
+    df = spark.sparkContext.parallelize(data) \
+            .toDF(["id", "name", "score"])
+
+    # 2. Write to starrocks with the format "starrocks",
+    # and replace the options with your own.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/93a433b8-46a7-4614-9ecb-c7ee31f557bc)
write "what", and your own "what"?